corehq/util/datadog/utils.py

```diff
@@ -115,7 +115,7 @@ def bucket_value(value, buckets, unit=''):
     lt_template = "lt_{:0%s}{}" % number_length
     over_template = "over_{:0%s}{}" % number_length
     for bucket in buckets:
-        if value < bucket:
+        if value <= bucket:
```

Comment:
Prometheus histograms uses `<=`. I changed this to make them compatible and to make testing easier but I can change if back. Note that I didn't change the tag name, it is still `lt` since that would break a bunch of DD graphs.

Reply:
Once it is merged, this will change the meaning of metrics recorded in dd, right? That's not ideal, but I suppose in the long run it will not matter. It's also not great to have the tag name `lt` mean something different than the recorded values.

Reply:
yes, that's correct. We could leave DD metrics as 'less than` and have Prometheus be `less than equal`, I would just need to update the tests to account for that.

Reply:
Updated to keep this as `<` for DD